ethcontract-common/src/bytecode.rs

```diff
@@ -60,9 +60,7 @@ impl Bytecode {
         S: AsRef<str>,
     {
         let name = name.as_ref();
-        if name.len() > 38 {
-            panic!("invalid library name for linking");
-        }
+        assert!(!(name.len() > 38), "invalid library name for linking");
```

Comment:
I know it was like this in the other branch, but maybe this shuold be:
```suggestion
        assert!(name.len() <= 38, "invalid library name for linking");
```